R/ggplot.R

```diff
@@ -167,9 +30,7 @@ ggplot_build_restore <- function() {
   }
 }
 
-
-
-
+# Careful, be aware that plotly uses this!
 ggthematic_build <- function(p, ggplot_build = .globals$ggplot_build, theme = .globals$theme) {
```

Comment:
Consider making this a S3 generic and exporting (with note about only for hooking custom ggplot objects into this behavior)

Reply:
This would've been great to have, but unfortunately doing it 'properly' on R 3.5 and below, so I've reverted it https://github.com/rstudio/thematic/pull/38/commits/3bda700077179ba60bc2a806b33e0ba13e77243f